Project Management Process.md

```diff
@@ -0,0 +1,293 @@
+# IPFS Project Management Process
+
+**Status: work in progress, not even a full draft yet**
+
+## Introduction
+TODO: describe the purpose of this document
+
+- this is an opt-in process, nobody in the community is *required* to use this process
+
+## Table on Contents
+- [TL;DR](#tldr)
+- [Projects](#projects)
+- [Roles](#roles)
+  - [Product Owners](#product-owners)
+  - [Project Leads](#project-leads)
+  - [Project Managers](#project-managers)
+- [Structure](#structure)
+  - [Overview](#overview)
+  - [Goals](#goals)
+  - [Backlog](#backlog)
+  - [Milestones](#milestones)
+  - [Project Roadmap](#project-roadmap)
+  - [Organization Roadmap](#organization-roadmap)
+- [Flow](#flow)
+  - [Quarterly Planning](#quarterly-planning)
+  - [Software Development Pipeline](#software-development-pipeline)
+    - [Kanban Pipeline](#kanban-pipeline)
+- [Implementation](#implementation)
+
+## TL;DR
+
+The IPFS project management process has the following **cadence**:
+
+Roadmap Updates
+- every 3 months
+- identify, discuss and plan the next steps for the organization as a whole
+
+"Sprint" Updates
+- once a week (2 weeks?)
+- discuss last week's progress
+- discuss plan for the next week
+
+The IPFS project management process breaks down to the following **units of work**:
+
+Projects 
+- may contain other projects
+- have a project leader who "roadmaps" (decides what are goals and milestones, and when they should happen)
+- have a roadmap (list of milestones and goals)
+- have a backlog (list of goals)
+
+Milestones 
+- measurable unit of progress
+- contains a list of goals, ordered by priority
```

Comment:
'Tasks' are notoriously bad units of work as they make you "do a thing". Goals frame the context differently by providing an end result to drive towards to, possibly including multiple 'tasks' to perform to reach it. Tasks are also way too micro-level to manage and they don't, there's no need to track things at such a granular level. When you reflect on progress and ask "what did we achieve this month?", it's easier to answer that question with goals than with tasks: "we improved the API documentation" vs. "we worked on the API documentation".

While we're not limited to "standard" terminology, it's also good to keep in mind that (all of?) these terms are standard ways to describe project management related activities and such, changing the meaning of the terms will be a knowledge barrier to entry for new projects and contributors.